packages/gatsby-remark-responsive-iframe/src/__tests__/index.js

```diff
@@ -73,6 +83,22 @@ describe(`gatsby-remark-responsive-iframe`, () => {
     })
   })
 
+  testInNode8OrHigher(`can copy JSX images`, async () => {
+    const mdx = require(`remark-mdx`)
+
+    const markdownAST = remark().use(mdx).parse(`
+<iframe url="http://www.example.com/" style="border:0;" width="600px" height="400px"></iframe>
+    `)
+
+    const transformed = await plugin({ markdownAST })
+    const node = find(transformed, function(node) {
+      return node.type === `html`
+    })
+
+    expect(node).toBeDefined()
+    expect(node.value).toMatchSnapshot()
```

Comment:
Yikes, it seems like snapshot assertion doesn't work well with `testInNode8OrHigher` (tests fail with `1 snapshot obsolete`)

Reply:
I'll update the test to not use a snapshot 👍